recipe/meta.yaml

```diff
@@ -9,9 +9,10 @@ source:
   sha256: 34f3fa2366ce20b18f1df573a77c1d27306ce1f2a41d9f9eff621b5192ea8788
   patches:
     - patches/use_system_blosc2.patch
+    - patches/0001-Don-t-copy-blosc2-in-the-package.patch
```

Comment:
why we do not want to copy it ?

Reply:
Because we should not vendor dependencies that we already have on defaults.

Reply:
yes, I can see now we have blosc-feedstock